build.sbt

```diff
@@ -50,7 +50,6 @@ lazy val scoverageSettings: Seq[Setting[_]] = Seq(
     "logger.*\\(.*\\)",
     // TODO: remove these as required as we build out the post reg group amendment
     ".*.controllers.amendment.group.AddGroupMemberController",
-    ".*.controllers.amendment.group.ConfirmRemoveMemberController",
     ".*.controllers.amendment.group.GroupMembersListController",
     ".*.controllers.amendment.group.ManageGroupMembersController",
     ".*.views.html.amendment.group.confirm_remove_member_page.*",
```

Comment:
will we add unit specs for the views in a future pr?

Reply:
Fair; I'll add the view test and stay opt'ed in.